libs/live-fns/keybase.js

```diff
@@ -0,0 +1,29 @@
+const got = require('../got.js')
+
+const getPGPFingerprint = user => {
+  const fingerprint = user.public_keys.primary.key_fingerprint
+
+  // Convert to 64-bit fingerprint
+  return fingerprint.substr(fingerprint.length - 16).toUpperCase().match(/.{1,4}/g).join(' ')
+}
+
+module.exports = async (topic, username) => {
+  const endpoint = `https://keybase.io/_/api/1.0/user/lookup.json?usernames=${username}`
```

Comment:
Use the Got [`query`](https://github.com/sindresorhus/got#query) option with an object. The current will probably work fine since it's just a username, but best practice and all.